src/types/fixed.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  isType (identifier) {
+    let mXn = identifier.substr(5)
+
+    if (!mXn || mXn.indexOf('x') === -1) return false
```

Comment:
Well, sorry to bother you again, but I was referring to the second condition only. An empty `mXn` should be valid, and actually, according to [this](https://solidity.readthedocs.io/en/develop/types.html#fixed-point-numbers) it should default to `128x18`. Could you do this little change?

Reply:
done :)